konan/konan.properties

```diff
@@ -61,6 +61,7 @@ osVersionMin.macos_x64 = 10.11
 entrySelector.macos_x64 = -alias _Konan_main _main
 dependencies.macos_x64 = \
     libffi-3.2.1-3-darwin-macos \
+    lldb-171-macos \
```

Comment:
Can you have consistent versioning?

Reply:
Bundles build queues are not synchronized currently. It is possible to hide this difference with artificial version numbers. I see two cons for this approach: 1) additional indirection layer which we would maintain 2) possible conflict with versioning for the builds in case we (as JB) would decide to publish them externally.